docs-ref-services/latest/speech-service.md

```diff
@@ -81,7 +81,8 @@ recognizer.recognizeOnceAsync(
   });
 ``` 
 
-Check out our [step-by-step quickstart](/azure/cognitive-services/speech-service/quickstart-js-node).
+
+The previous example uses single-shot recognition, which recognizes a single utterance. You can also use [continuous recognition](/azure/cognitive-services/speech-service/get-started-speech-to-text?tabs=windowsinstall&pivots=programming-language-nodejs#continuous-recognition) to control when to stop recognizing. Check out our [step-by-step quickstart](/azure/cognitive-services/speech-service/quickstart-js-node) for many more options.
```

Comment:
Unfortunately, these links don't resolve on the review site (because of the fact that they go to a different repo source that doesn't have the review branch. When I manually adjust the destination links to non-review, they check out.

https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/quickstart-js-node
https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/get-started-speech-to-text?tabs=windowsinstall&pivots=programming-language-nodejs#continuous-recognition